### PR TITLE
Fixes in overlayed mode overflow issue in chat box in home page

### LINKED ISSUE #3133: In overlayed mode the input form icons scroll with the page
**Describe the bug**
In overlayed mode, scrolling causes the input icons on the login form to scroll outside the inputs

**To Reproduce**
1. Go to converse.js
2. Click on Toggle to open the chat
3. Move mouse to main page and scroll
4. See icons move on the chat window form

(notice that they also scroll off the inputs, but stay on the form, if you ONLY scroll the form itself and not just the page)

**Expected behavior**
Icons should stay inside the inputs

**Video**

https://user-images.githubusercontent.com/1156100/216807126-bbf09c6c-7d3e-499f-b67b-900b8e1249d2.mp4

**Environment (please complete the following information):**
 - Desktop with overlayed mode
 - Browser Chrome latest
 - Converse.js 8

### PR DESCRIPTION
**Pull request by love-sonkar**
Fixes #3133 issue changes overflow-y to none

Thanks for making a pull request to converse.js!

Before submitting your request, please make sure the following conditions are met:

- [Y] Add a changelog entry for your change in `CHANGES.md`
- [Y] When adding a configuration variable, please make sure to
      document it in `docs/source/configuration.rst`
- [Y] Please add a test for your change. Tests can be run in the commandline
      with `make check` or you can run them in the browser by running `make serve`
      and then opening `http://localhost:8000/tests.html`.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 Breaking changes:
 - Remove the old `_converse.BootstrapModal` in favor of `_converse.BaseModal` which is a web component.
+- #fixes #3133 changes _controlbox.scss to fix the overflow issue
```
Comment (jcbrand) [inline CHANGES.md:7]: This isn't a breaking change, so shouldn't go under the `Breaking changes` heading.

File: src/plugins/controlbox/styles/_controlbox.scss [hunk 2]
```diff
@@ -53,6 +53,7 @@
     #controlbox {
         order: -1;
         color: var(--controlbox-text-color);
+        overflow-y:none;
```
Comment (jcbrand) [inline src/plugins/controlbox/styles/_controlbox.scss:56]: Is `none` a valid value?

It's not shown here as one: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y

What's the effect of this? Does it disable vertical scrolling in the controlbox?
If so, I'm not convinced that this is the right solution, since the controlbox might go off-screen on certain displays.

Did you check what happens in fullscreen mode? Might be that this bug should only be fixed for overlayed view mode, in which case you need to put any CSS changes under the following:

```
.conversejs {
    converse-chats {
        &.converse-overlayed {
            #controlbox {
```

File: src/plugins/controlbox/styles/controlbox-head.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@
             flex-direction: row-reverse;
             flex-wrap: nowrap;
             justify-content: space-between;
-            min-height: 0;
+            min-height: 30px;
```
Comment (jcbrand) [inline src/plugins/controlbox/styles/controlbox-head.scss:8]: Why was this necessary?

**Review by jcbrand — COMMENTED** [review]
Thanks for your pull request. I've left some review comments.

File: CHANGES.md (hunk 1 above)
Comment (love-sonkar) [reply CHANGES.md:7]: okay i'm new here so i don't know where i put those changes list. can you tell me whare i write these changes

File: src/plugins/controlbox/styles/_controlbox.scss (hunk 2 above)
Comment (love-sonkar) [reply src/plugins/controlbox/styles/_controlbox.scss:56]: yeah it's valid value it's work like hidden property but i can changes to the hidden for convinence,

i found a one more thing while checking the chat box there is a dragabble property, and we can resize the chat box but we should stop the resize property to a specific height so we can set the flyout min-height to 500px so that when user try to resize this to 400px it won't be an we can't see any scrollbar this is my approch to solve this bug.

File: src/plugins/controlbox/styles/controlbox-head.scss (hunk 3 above)
Comment (love-sonkar) [reply src/plugins/controlbox/styles/controlbox-head.scss:8]: sorry i haven't check well i forgot to check properly now it's not needed to set min-height to 30px because the chat box is resizable box so it won't affect

File: CHANGES.md (hunk 1 above)
Comment (jcbrand) [reply CHANGES.md:7]: Just move it up above the `Breaking changes` heading (but below `# 11.0.0 (Unreleased)`).

File: src/plugins/controlbox/styles/_controlbox.scss (hunk 2 above)
Comment (jcbrand) [reply src/plugins/controlbox/styles/_controlbox.scss:56]: > i can changes to the hidden for convinence

Yes, `hidden` would be better.

**Comment by love-sonkar** [comment]
hey sir i have fixed the issue as you told me

File: src/plugins/controlbox/styles/_controlbox.scss (hunk 2 above)
Comment (jcbrand) [reply src/plugins/controlbox/styles/_controlbox.scss]: I've thought about this some more, and I don't think this is the right solution.

You're simply disabling scrolling, which means that if the contents would overflow, it would be invisible and the user cannot scroll to see it.

Also, since I don't know where those icons reported in #3133 come from, it's not clear to me what needs to be fixed, how it should be done or whether there really is anything to fix.

**Comment by jcbrand** [comment]
Closing based on https://github.com/conversejs/converse.js/pull/3272#discussion_r1341013375

Thanks for the effort.
